huffman/test/bit_span_test.cpp

```diff
@@ -84,12 +83,71 @@ auto main() -> int
   using ::boost::ut::operator|;
 
   test("aborts if bit offset too large") = [](auto bit_offset) {
-    static constexpr auto data =
-        std::array{std::byte{0b10101010}, std::byte{0xff}};
+    static constexpr auto data = huffman::byte_array(0b10101010, 0xff);
 
     expect(aborts([&] {
       static constexpr auto bit_size = 7;
       huffman::bit_span{data.begin(), bit_size, bit_offset};
     }));
   } | std::vector<std::uint8_t>{8, 9, 10};  // NOLINT(readability-magic-numbers)
+
+  std::vector<unsigned> n_to_consume(1Z + 2Z * CHAR_BIT);
+  std::iota(n_to_consume.begin(), n_to_consume.end(), 0);
```

Comment:
Do either of these work?

```cpp
const auto ... = std::views::iota(0UZ, 1UZ + 2UZ * CHAR_BIT) | std::ranges::to<std::vector>();
```

https://en.cppreference.com/w/cpp/ranges/to

```cpp
const auto ... = std::vector(std::from_range, std::views::iota(0UZ, 1UZ + 2UZ * CHAR_BIT));
```

https://en.cppreference.com/w/cpp/ranges/from_range